src/linux/backend-v4l2.h

```diff
@@ -413,6 +416,12 @@ namespace librealsense
             std::atomic<bool> _is_started;
             std::unique_ptr<std::thread> _thread;
             std::unique_ptr<named_mutex> _named_mtx;
+            struct device {
+                enum v4l2_buf_type buf_type;
+                unsigned char num_planes;
+                struct v4l2_capability cap;
+                struct v4l2_cropcap cropcap;
```

Comment:
please add some comment to explain the meaning of cropcap

Reply:
there is no function for LRS but it was polled this device capability in past. This is "crop capability" for frame cropping.